docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -98,6 +98,58 @@ strapi watch-admin
 options: [--browser <name>]
 ```
 
+
+## strapi export <BetaBadge />
+
+Exports your project data. The default settings create a `.tar` file, compressed using `gzip` and encrypted using `aes-128-ecb`.
+
+```bash
+strapi export
+```
+
+The exported file is automatically named using the format `export_YYYYMMDDHHMMSS` with the current date and timestamp. Alternately, you can specify the filename using the `-f` or `--file` flag. The following table provides all of the available options as command line flags:
+
+| Option           | Type    | Description                                                                                               |
+|------------------|---------|----------------------------------------------------------------------------|
+| `--no-encrypt`     |         | Disables file encryption and disables the `key` option.                                                   |
+| `--no-compress`    |         | Disables file compression.                                                                                |
+| `-k`, `--key`            | string  | Passes the encryption key as part of the `export` command. <br> The `--key` option can't be combined with `--no-encrypt`. |                                |
+| `-f`, `--file`       | file    | Specifies the export filename. Do not include a file extension.                                           |
+| `-h`, `--help`       |         | Displays help for the `strapi export` command.                                                            |
+
+<!--| --exclude        | string  | Comma-separated list of data to exclude (files [localMediaFiles, providerMediaFiles], content [entities, links], schema, configuration). |         |  example: strapi export --exclude XXXXXX #exports your data excluding XXXXX.-->
+
+**Examples**
+
+```bash
+#examples of strapi export:
+
+strapi export -f myData #exports your data with the default options and the filename myData.
```

Comment:
```suggestion
strapi export -f myData #exports your data with the default options and the filename myData (which will result in a file named myData.tar.gz.enc)
```
Not the best wording on my part, but maybe something like this would help